codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -185,6 +195,22 @@ public final T build(Http2Connection connection) {
                 reader = new Http2InboundFrameLogger(reader, frameLogger);
                 writer = new Http2OutboundFrameLogger(writer, frameLogger);
             }
+
+            // Configure the local flow controller.
+            if (connection.local().flowController() == null) {
+                connection.local()
+                        .flowController(new DefaultHttp2LocalFlowController(connection,
+                                DEFAULT_WINDOW_UPDATE_RATIO, autoRefillConnectionWindow));
```

Comment:
Hmm ... maybe get rid of this setting altogether, now that we can correctly specify the inbound flow controller in the connection?  If not specified, we'll create a default with the autorefill = false.  WDYT?